### PR TITLE
New alternative routes UI

### PR DESCRIPTION
**Pull request by Wouter125**
- [x] New route with alternatives within the EcoMovement boundaries
- [x] New UI that reflects the updated design
- [x] Patched documentation

### CODE REVIEW

File: alternative-routes/index.js [hunk 1]
```diff
@@ -115,39 +119,67 @@ const decodePolylines = (route, alternatives) => {
...
-  // the total energy used of the route, in kWh
-  document.getElementById('consumption-overview').innerHTML = data.consumption
-    ? `${data.consumption.toFixed(2)} kWh`
-    : 'Unknown';
-  document.getElementById('consumption').innerHTML = data.consumption
-    ? `${data.consumption.toFixed(2)} kWh`
-    : 'Unknown';
+  document.getElementById('duration').innerHTML = `${getDurationString(data.duration ?? 0)}`;
+  document.getElementById('route-metadata').innerHTML = `${routeDistance} / ${routeStops} / ${routeEnergy}`;
+};
```
Comment (lsmienk) [inline alternative-routes/index.js:149]: The route header should also be updated when the user switches to an alternative route.

**Review by lsmienk — CHANGES_REQUESTED** [review]
Looks good to me, I think it would be good to have the data on the header also change.

File: alternative-routes/index.js (hunk 1 above)
Comment (Wouter125) [reply alternative-routes/index.js:149]: Yup, missed that one. Good catch.

**Comment by Wouter125** [comment]
Patched the issue, if you are okay with it you can merge it.

**Review by lsmienk — APPROVED** [review]

**Pull request merged**